downstream/modules/eda/proc-modifying-activations-after-install.adoc

```diff
@@ -3,7 +3,7 @@
 = Modifying the number of simultaneous rulebook activations after {EDAcontroller} installation
 
 [role="_abstract"]
-By default, {EDAcontroller} allows 12 activations to run simultaneously.
+By default, {EDAcontroller} allows 12 rulebook activations per node. For example, with two worker or hybrid nodes, it results in a limit of 24 activations in total to run simultaneously. 
```

Comment:
I think we should further clarify that this works slightly different on OCP like mentioned [here](https://github.com/ansible/aap-docs/pull/2602#discussion_r1850799022). We should state that for OCP this is a global value, but for containerized installs, this value is per node

Reply:
Actually, we have that part covered in the section above this sub-section. It's stated in the 2nd bullet point of the **Note** admonition in [10.1.1. Modifying the number of simultaneous rulebook activations](https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html-single/using_automation_decisions/index#modifying-simultaneous-activations). 

Then the section we modified above follows that. So there's 10.1.1.1. Modifying the number of simultaneous rulebook activations during Event-Driven Ansible controller installation and 10.1.1.2. Modifying the number of simultaneous rulebook activations after Event-Driven Ansible controller installation that tells the user how to modify the activation limits. We could add the same note after each sub-section, but that seems somewhat superfluous to me when we cover it in the high-level chapter before these two sections. Let me know your thoughts. Also, take a look at the entire section and how it flows currently.

Reply:
Sounds good, I see that now. Thank you Jameria